zha/application/platforms/update.py

```diff
@@ -284,7 +281,7 @@ async def async_install(self, version: str | None) -> None:
                 raise ZHAException(f"Version {version!r} is not available")
 
         self._attr_in_progress = True
-        self._attr_progress = 0
+        self._attr_update_percentage = None
```

Comment:
This is a change in semantics: when an update starts, we don't have any progress information and show an indeterminate progress bar. This is because devices can sometimes resume an old update halfway through.